web/src/features/moreCast2/components/MoreCast2Column.tsx

```diff
@@ -152,28 +155,95 @@ export class IndeterminateField implements ColDefGenerator, ForecastColDefGenera
+        columnClickHandlerProps,
+        headerName ?? this.headerName,
+        this.preProcessEditCellProps
+      )
     }
   }
 
   public generateForecastSummaryColDef = (columnClickHandlerProps: ColumnClickHandlerProps) => {
-    return this.colDefBuilder.generateForecastColDef(columnClickHandlerProps)
+    return this.colDefBuilder.generateForecastColDef(columnClickHandlerProps, undefined, this.preProcessEditCellProps)
   }
 
   public generateColDef = () => {
-    return this.colDefBuilder.generateColDefWith(this.field, this.headerName, this.precision)
+    return this.colDefBuilder.generateColDefWith(
+      this.field,
+      this.headerName,
+      this.precision,
+      undefined,
+      this.preProcessEditCellProps
+    )
   }
 
   public generateColDefs = (columnClickHandlerProps: ColumnClickHandlerProps, headerName?: string) => {
-    return this.colDefBuilder.generateColDefs(columnClickHandlerProps, headerName ?? this.headerName, this.includeBias)
+    return this.colDefBuilder.generateColDefs(
+      columnClickHandlerProps,
+      headerName ?? this.headerName,
+      this.includeBias,
+      this.preProcessEditCellProps
+    )
   }
 }
 
-export const tempForecastField = new IndeterminateField('temp', TEMP_HEADER, 'number', 0, true)
-export const rhForecastField = new IndeterminateField('rh', RH_HEADER, 'number', 0, true)
-export const windDirForecastField = new IndeterminateField('windDirection', WIND_DIR_HEADER, 'number', 0, true)
-export const windSpeedForecastField = new IndeterminateField('windSpeed', WIND_SPEED_HEADER, 'number', 0, true)
-export const precipForecastField = new IndeterminateField('precip', PRECIP_HEADER, 'number', 1, true)
+export const tempForecastField = new IndeterminateField(
+  'temp',
+  TEMP_HEADER,
+  'number',
+  0,
+  true,
+  (params: GridPreProcessEditCellProps) => {
+    const error = params.props.value < -60 || params.props.value > 60 ? 'Temp must be between -60 and 60 degrees C' : ''
+    return { ...params.props, error }
+  }
+)
+export const rhForecastField = new IndeterminateField(
+  'rh',
+  RH_HEADER,
+  'number',
+  0,
+  true,
+  (params: GridPreProcessEditCellProps) => {
+    const error = params.props.value < 0 || params.props.value > 100 ? 'RH must be between 0 and 100' : ''
+    return { ...params.props, error }
+  }
+)
+export const windDirForecastField = new IndeterminateField(
+  'windDirection',
+  WIND_DIR_HEADER,
+  'number',
+  0,
+  true,
+  (params: GridPreProcessEditCellProps) => {
+    const error =
+      params.props.value < 0 || params.props.value > 360 ? 'Wind direction must be between 0 and 360 degrees' : ''
+    return { ...params.props, error }
+  }
+)
+export const windSpeedForecastField = new IndeterminateField(
+  'windSpeed',
+  WIND_SPEED_HEADER,
+  'number',
+  0,
+  true,
+  (params: GridPreProcessEditCellProps) => {
+    const error =
+      params.props.value < 0 || params.props.value > 120 ? 'Wind speed must be between 0 and 120 degrees' : ''
```

Comment:
```suggestion
      params.props.value < 0 || params.props.value > 120 ? 'Wind speed must be between 0 and 120 kph' : ''
```

Reply:
Fixed in: https://github.com/bcgov/wps/pull/3919/commits/2d168d46afd8553718057507c0268f3553f6c2e9